database/src/adapters/aggregated/circulating_supply.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) async fn add_circulating_supply(
             }
             Err(async_error) => {
                 error!(
-                    target: crate::AGGREGATED,
+                    target: crate::EXPLORER_DATABASE,
```

Comment:
I suggest getting rid of using `explorer` word in indexer projects.
Historically, Explorer was the main and the only user for the DB, now it's not true.
What do you think about just `INDEXER_DATABASE`?

Reply:
@telezhnaya but it is Explorer Database and we call it so. Let's keep it, at least as a tradition for this project. :)

Reply:
Agree with @khorolets here, we're writing to the `explorer` database so this naming makes sense?

Reply:
Decision: we keep the Explorer mentions as this project and its database serves the needs of Explorer in the first place.